pkg/artifactory/resource_artifactory_user_test.go

```diff
@@ -73,9 +73,9 @@ func TestAccUserPasswordNotChangeWhenOtherAttributesChangeGH340(t *testing.T) {
 func TestAccUser_basic(t *testing.T) {
 	const userBasic = `
 		resource "artifactory_user" "%s" {
-			name  	= "the.dude%d"
+			name  	= "dummy_user%d"
```

Comment:
You don't like?!

Reply:
Just trying to be consistent with the other tests 😄